src/Front/DesugarLists.hs

```diff
@@ -4,6 +4,7 @@ module Front.DesugarLists
   ) where
 
 import qualified Common.Compiler               as Compiler
+import Common.Identifiers                       (Identifier(Identifier))
```

Comment:
Run the formatter over this DesugarLists.hs file (should be able to do this in VS code via the command palette). Follow up if you are unsure how to do this.